.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
+<!--
+
+Currently nixpkgs has a lot of new incoming PR's, but not nearly enough people to review this constant stream. Even if you aren't a committer, we would appreciate a review from you, especially for package updates. Just testing the relevant package/service and leaving a comment what you tested and whether it worked would be great. See https://github.com/NixOS/nixpkgs/pulls for open PR's. Reviewing PR's isn't mandatory of course, but it would help out the committers immensely and make the average time-to-merge shorter for all of us. Thanks a lot if you do!
```

Comment:
Do we have a reviewers template for package updates?

Defining a set of things to do and what to look for can make it even clearer for someone to just jump in and start reviewing.

We will also get across what kind of quality we expect from reviewers.